--- gedit-3.36.1/meson.build.orig	2020-04-16 18:16:08.510741221 +0200
+++ gedit-3.36.1/meson.build	2020-04-16 18:21:02.075817509 +0200
@@ -31,7 +31,7 @@
 pkgdatadir = join_paths(datadir, meson.project_name())
 
 glibdir = join_paths(datadir, 'glib-2.0')
-gtkdocdir = join_paths(datadir, 'gtk-doc')
+gtkdocdir = join_paths(datadir, 'doc', 'gtk-doc')
 vapidir = join_paths(datadir, 'vala', 'vapi')
 
 # Subprojects
