--- gedit-2.17.4/data/gedit.desktop.in.in~	2007-01-30 10:59:39.000000000 +0100
+++ gedit-2.17.4/data/gedit.desktop.in.in	2007-01-30 11:01:31.287208866 +0100
@@ -8,7 +8,7 @@
 StartupNotify=true
 MimeType=text/plain;
 Icon=accessories-text-editor
-Categories=GNOME;GTK;Application;Utility;TextEditor;
+Categories=GTK;GNOME;Utility;TextEditor;
 X-GNOME-DocPath=gedit/gedit.xml
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gedit
