diff -Nuard gedit-2.4.1.orig/data/gedit.schemas.in gedit-2.4.1/data/gedit.schemas.in
--- gedit-2.4.1.orig/data/gedit.schemas.in	2003-06-12 19:59:58.000000000 +0200
+++ gedit-2.4.1/data/gedit.schemas.in	2004-02-24 22:40:36.668714456 +0100
@@ -5,7 +5,7 @@
       <applyto>/apps/gedit-2/preferences/editor/font/use_default_font</applyto>
       <owner>gedit</owner>
       <type>bool</type>
-      <default>FALSE</default>
+      <default>TRUE</default>
       <locale name="C">
 	<short>Use Default Font</short>
 	<long>Whether to use the system's default font for editing text
