diff -urN gedit-2.22.3/configure.ac gedit-2.22.3.new/configure.ac
--- gedit-2.22.3/configure.ac	2008-05-27 14:05:34.000000000 +0200
+++ gedit-2.22.3.new/configure.ac	2008-05-28 17:58:51.000000000 +0200
@@ -31,6 +31,8 @@
 IT_PROG_INTLTOOL([0.35.0])
 
 AC_PROG_LIBTOOL
+dnl when using libtool 2.x create libtool early, because it's used in configure
+m4_ifdef([LT_OUTPUT], [LT_OUTPUT])
 
 GNOME_DOC_INIT
 GTK_DOC_CHECK([1.0])
