diff -urN gedit-2.9.7.orig/data/gedit.desktop.in gedit-2.9.7/data/gedit.desktop.in
--- gedit-2.9.7.orig/data/gedit.desktop.in	2005-03-01 13:37:50.346344976 +0100
+++ gedit-2.9.7/data/gedit.desktop.in	2005-03-01 13:38:28.449552408 +0100
@@ -8,7 +8,7 @@
 StartupNotify=true
 MimeType=text/plain;
 Icon=text-editor
-Categories=Application;Utility;TextEditor;
+Categories=GTK;GNOME;Utility;TextEditor;
 X-GNOME-DocPath=gedit/gedit.xml
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gedit
