diff -urN gedit-2.5.91.orig/configure.in gedit-2.5.91/configure.in
--- gedit-2.5.91.orig/configure.in	2004-03-09 11:51:48.173618408 +0100
+++ gedit-2.5.91/configure.in	2004-03-09 11:52:19.642834360 +0100
@@ -30,7 +30,7 @@
 GETTEXT_PACKAGE=gedit-2.6
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE", [Gettext package])
-ALL_LINGUAS="am ar be bg bn az ca cs cy da de el en_CA es et eu fa fi fr ga gl he hi hr hu it ja kn ko lt lv mi mk ml mn ms ne nl nn no pa pl pt pt_BR ro ru sv sk sl sq sr sr@Latn ta th tr uk vi wa zh_CN zh_TW"
+ALL_LINGUAS="am ar be bg bn az ca cs cy da de el en_CA es et eu fa fi fr ga gl he hi hr hu it ja kn ko lt lv mi mk ml mn ms nb ne nl nn pa pl pt pt_BR ro ru sv sk sl sq sr sr@Latn ta th tr uk vi wa zh_CN zh_TW"
 AM_GLIB_GNU_GETTEXT
 
 dnl ******************
